docs/docsite/rst/shared_snippets/variable_evaluation.rst

```diff
@@ -0,0 +1,30 @@
+.. _variable_evaluation:
+
+Variable Evaluation
```

Comment:
My impression is that this topic is more about the lazy evaluation of **templates** rather than variables